### ISSUE TITLE
Argument "4.2.1" isn't numeric in numeric ge (>=) at ./scripts/feeds line 25

### ISSUE THREAD

**Issue by hexchain** [openwrt/openwrt#24]
`make -v`:
```
GNU Make 4.2.1
Built for x86_64-pc-linux-gnu
Copyright (C) 1988-2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

```
$mkver[2] >= "3.81" or $valid_mk = 0;
```
This line assumes `mkver[2]` to be a number, which fails in this case.

**Comment by ChaoticMind** [comment]
This was fixed in f51dd01.

**Comment by hexchain** [comment]
Confirmed. Thank you!
